ajax/libs/colors/package.json

```diff
@@ -1,23 +1,39 @@
 {
   "name": "colors",
-  "filename": "colors.min.css",
-  "version": "1.0",
-  "description": "Better colors for the web. A collection of skin classes for faster prototyping and nicer looking sites.",
+  "version": "3.0.0",
+  "filename": "js/colors.js",
```

Comment:
After you minify the file, lets's use `color.min.js` instead of `color.js` in the filename field.